tracer/build/supported_versions.yml

```diff
@@ -0,0 +1,1071 @@
+[
```

Comment:
This file's name has a `yml` extension but its contents sure looks like JSON to me 😅

Reply:
[JSON is valid yaml don't cha' know](https://til.codeinthehole.com/posts/json-is-valid-yaml/). But yeah. Oops. 😂

Reply:
TIL!